backend/conf/demo.conf

```diff
@@ -23,4 +23,9 @@ akka.contrib.persistence.mongodb.mongo.mongouri = "mongodb://localhost:27017/las
 akka.contrib.persistence.mongodb.mongo.mongouri = ${?MONGODB_URI}
 
 lasius.title = Lasius
-lasius.instance = Demo
\ No newline at end of file
+lasius.instance = Demo
+
+lasius.plane.api_key = "plane_api_08ba10840ad34801baab20fba68a784c"
```

Comment:
oh, oh...that's a pre-christmas gift ;-)

Reply:
noooo!